packages/cli/src/Bootstrap.ts

```diff
@@ -72,22 +72,52 @@ export class Bootstrap {
       bootstrap = { ...bootstrap, ...currentBootstrap };
     }
     const kernel = bootstrap.kernel();
+
     await kernel.boot();
     const serviceProviders = bootstrap.serviceProviders;
     
     for (const serviceProvider of serviceProviders) {
       await kernel.registerServiceProvider(serviceProvider);
     }
     
+    let transport;
+
     if (command !== 'cli' && (command in bootstrap.transport)) {
-      const transport = bootstrap.transport[command](kernel);
+      transport = bootstrap.transport[command](kernel);
       await transport.up(opts);
-      return transport;  
+    } else {
+      transport = bootstrap.transport.cli(kernel);
+      await transport.up(argv);
+    }
+
+    this.registerShutdownHook(kernel, transport);
+    
+    return transport;  
+  }
+
+  registerShutdownHook(kernel: Interfaces.KernelInterface, transport: Interfaces.TransportInterface) {
+    function handle() {
+      setTimeout(() => {
+        process.exit(0);
+      }, 5000);
+
+      transport.down()
+        .then(() => {
+          kernel.shutdown()
+            .then(() => {
+              process.exit(1);
```

Comment:
exit(1) --> smth wrong
exit(0) --> all OK!

need to swap ?

Reply:
Right :)